packages/ipython.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,7 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'ipython'
+$category = 'Productivity Tools'
```

Comment:
Do we need a shortcut for ipython in the Productivity Tools folder? I personally think we don't and that we should directly use `VM-Pip-Install ipython` instead. But what do others think? @mr-tz @mandiant/vms opinions?

Reply:
@Ana06 
By default, ipython is installed and available from the command line (I believe it is a dependency for another installed python library). This package installs its shortcut under `Productivity Tools`.

Reply:
You should always test new packages in an empty VM as documented in https://github.com/mandiant/VM-Packages/wiki/Testing#local-testing. The users can select the packages to install when installing FLARE-VM and commando VM uses different packages as well. In a VM with FLARE-VM and only the following packages I do not have ipython available:
```
Boxstarter|3.0.3
Boxstarter.Bootstrapper|3.0.3
Boxstarter.Chocolatey|3.0.3
Boxstarter.Common|3.0.3
Boxstarter.HyperV|3.0.3
Boxstarter.WinConfig|3.0.3
chocolatey|2.3.0
chocolatey-compatibility.extension|1.0.0
chocolatey-core.extension|1.4.0
chocolatey-windowsupdate.extension|1.0.5
common.vm|0.0.0.20240913
debloat.vm|0.0.0.20240327
hashmyfiles.vm|0.0.0.20240411
installer.vm|0.0.0.20240402
KB2919355|1.0.20160915
KB2919442|1.0.20160915
KB2999226|1.0.20181019
KB3033929|1.0.5
KB3035131|1.0.3
pma-labs.vm|0.0.0.20240411
python3|3.10.11
python3.vm|0.0.0.20240726
python310|3.10.11
vcredist140|14.40.33810
vcredist2015|14.0.24215.20170201
```

I hadn't noticed that the default configuration installs ipython. I wonder which package is installing it. Any clues @mandiant/vms?

Reply:
@d35ha Is the shortcut useful? 

I personally haven't used iPython directly, I think only as part of Jupyter Notebooks, so I'm not too familiar with how often it gets used directly.

Reply:
I normally use it, it is a way better than the traditional python shell.

Reply:
@d35ha I assume you mean you use IPython and not the shortcut. I personally think IPython does not need a shortcut and that this package should only install it.